src/sentry/incidents/subscription_processor.py

```diff
@@ -398,12 +398,15 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
             has_anomaly_detection
             and self.alert_rule.detection_type == AlertRuleDetectionType.DYNAMIC
         ):
-            potential_anomalies = get_anomaly_data_from_seer(
-                alert_rule=self.alert_rule,
-                subscription=self.subscription,
-                last_update=self.last_update.timestamp(),
-                aggregation_value=aggregation_value,
-            )
+            with metrics.timer(
```

Comment:
Isn't metrics getting deprecated? I thought we should just stick to traces.

Reply:
This is for Datadog, I haven't heard that we're getting rid of it. Since we have a metrics timer when the task is called [here](https://github.com/getsentry/sentry/blob/master/src/sentry/incidents/tasks.py#L175) I thought it'd be useful to use the same product so we can see how long this part takes compared to the whole time. I could move both to using traces though.